scipy/optimize/_optimize.py

```diff
@@ -1293,6 +1308,9 @@ def fmin_bfgs(f, x0, fprime=None, args=(), gtol=1e-5, norm=np.inf,
         Parameter for Armijo condition rule.
     c2 : float, default: 0.9
         Parameter for curvature condition rule.
+    hess_inv0 : None or ndarray, optional
+        Initial inverse hessian estimate. If None (default) then the identity
```

Comment:
State shape that the ndarray should have?

Reply:
The shape of `x0` is not documented in any `fmin_*` or `_minimize_*` function. Shall I add it at least for `_minimize_bfgs` as the shape of the Hessian will depend on it?